disnake/ext/commands/context.py

```diff
@@ -348,7 +348,7 @@ async def send_help(self, *args: Any) -> Any:
             return None
 
         try:
-            entity.qualified_name
+            entity.qualified_name  # noqa: B018
         except AttributeError:
             # if we're here then it's not a cog, group, or command.
             return None
```

Comment:
this should just use `hasattr`

Reply:
[`a071fd6` (#999)](https://github.com/DisnakeDev/disnake/pull/999/commits/a071fd6f15a8b5b540a7e605b46f99cd23ac86c6)